src/test/java/com/example/demo/service/mapper/ProductMapperTest.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.service.mapper;
+
+import com.example.demo.model.Product;
+import com.example.demo.model.ReviewDto;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ProductMapperTest {
+    private static ReviewDto REVIEW_DTO;
```

Comment:
this is not a constant field because it is not final
same for other test classes

Reply:
done